Exec/RegTests/SootRadTest/pelelmex_prob.H

```diff
@@ -0,0 +1,140 @@
+  pele::physics::PMF::pmf(pmf_data, x1, x2, pmf_vals);
+  state(i, j, k, TEMP) = pmf_vals[1];
+  amrex::Real norm = 0.;
+  for (int n = 0; n < NUM_SPECIES; n++) {
+    massfrac[n] = amrex::max(0., amrex::min(1., pmf_vals[3 + n]));
+    norm += massfrac[n];
+  }
+  for (int n = 0; n < NUM_SPECIES; ++n) {
+    massfrac[n] = massfrac[n] / norm;
+  }
+  AMREX_D_TERM(state(i, j, k, VELX) = pmf_vals[0] * 1.E-2;
+               , state(i, j, k, VELY) = 0.;, state(i, j, k, VELZ) = 0.;);
+  amrex::Real rho_cgs, P_cgs;
+  P_cgs = prob_parm.P_mean * 10.;
+
+  auto eos = pele::physics::PhysicsType::eos();
+  eos.PYT2R(P_cgs, massfrac.data(), state(i, j, k, TEMP), rho_cgs);
+  state(i, j, k, DENSITY) = rho_cgs * 1.0e3; // CGS -> MKS conversion
+
+  eos.TY2H(state(i, j, k, TEMP), massfrac.data(), state(i, j, k, RHOH));
+  state(i, j, k, RHOH) = state(i, j, k, RHOH) * 1.0e-4 *
+                         state(i, j, k, DENSITY); // CGS -> MKS conversion
+
+  for (int n = 0; n < NUM_SPECIES; n++) {
+    state(i, j, k, FIRSTSPEC + n) = massfrac[n] * state(i, j, k, DENSITY);
+  }
+  for (int is = 0; is < NUM_SOOT_MOMENTS + 1; ++is) {
+    state(i, j, k, FIRSTSOOT + is) = prob_parm.soot_vals[is];
+  }
+}
+
+AMREX_GPU_DEVICE
+AMREX_FORCE_INLINE
+void
+bcnormal(
+  const amrex::Real x[AMREX_SPACEDIM],
+  const int /*m_nAux*/,
+  amrex::Real s_ext[NVAR],
+  const int idir,
+  const int sgn,
+  const amrex::Real time,
+  amrex::GeometryData const& geomdata,
+  ProbParm const& prob_parm,
+  pele::physics::PMF::PmfData::DataContainer const* pmf_data)
+{
+  const amrex::Real* prob_lo = geomdata.ProbLo();
+  amrex::GpuArray<amrex::Real, NUM_SPECIES + 4> pmf_vals = {{0.0}};
+  amrex::Real massfrac[NUM_SPECIES] = {0.0};
+  if (sgn == 1) {
+    pele::physics::PMF::pmf(pmf_data, prob_lo[idir], prob_lo[idir], pmf_vals);
+    AMREX_D_TERM(s_ext[VELX] = pmf_vals[0] * 1.E-2;, s_ext[VELY] = 0.0;
+                 , s_ext[VELZ] = 0.0;);
+
+    s_ext[TEMP] = pmf_vals[1];
+
+    for (int n = 0; n < NUM_SPECIES; n++) {
+      massfrac[n] = pmf_vals[3 + n];
+    }
+
+    amrex::Real rho_cgs, P_cgs, RhoH_temp;
+    P_cgs = prob_parm.P_mean * 10.0;
+
+    auto eos = pele::physics::PhysicsType::eos();
+    eos.PYT2R(P_cgs, massfrac, s_ext[TEMP], rho_cgs);
+    s_ext[DENSITY] = rho_cgs * 1.0e3;
+
+    eos.TY2H(s_ext[TEMP], massfrac, RhoH_temp);
+    s_ext[RHOH] = RhoH_temp * 1.0e-4 * s_ext[DENSITY]; // CGS -> MKS conversion
+
+    for (int n = 0; n < NUM_SPECIES; n++) {
+      s_ext[FIRSTSPEC + n] = massfrac[n] * s_ext[DENSITY];
+    }
+    for (int is = 0; is < NUM_SOOT_MOMENTS + 1; ++is) {
+      s_ext[FIRSTSOOT + is] = prob_parm.soot_vals[is];
+    }
+  }
+}
+
+AMREX_GPU_DEVICE
+AMREX_FORCE_INLINE
+void
+zero_visc(
+  int i,
+  int j,
+  int k,
+  amrex::Array4<amrex::Real> const& beta,
+  amrex::GeometryData const& geomdata,
+  amrex::Box const& domainBox,
+  const int dir,
+  const int beta_comp,
+  const int nComp)
+{
+  amrex::ignore_unused(
```

Comment:
No need for this, just comment out the var names in the function arguments.

Reply:
Going to leave this here because this function uses ignore_unused in all the PeleLMeX cases